.github/workflows/development.yml

```diff
@@ -107,8 +109,9 @@ jobs:
       with:
         inlineScript: |
           az containerapp update \
-          --name ca-mdm-${{ env.product }}-${{ env.environment }}-001 \
+          --name ca-${{ env.product }}-mdm-${{ env.environment }}-001 \
           --image ${{ env.acr }}/${{ env.name }}:${{ env.from }} \
+          --revision-suffix v${{ github.run_number }} \
```

Comment:
I'd usually use the git commit hash as a suffix - is it going to be easy enough to tell which commit of the code is deployed to the container app?

Reply:
Its quite common to use truncated commit hash. In Mulesoft deployment action we use
commitHash=$(git rev-parse --short "$GITHUB_SHA")

GITHUB_SHA is populated by github action